DigitalLearningSolutions.Web/ViewModels/TrackingSystem/EditRegistrationPromptViewModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+{
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Data.Models.CustomPrompts;
+    using DigitalLearningSolutions.Web.Helpers;
+
+    public class EditRegistrationPromptViewModel
+    {
+        public EditRegistrationPromptViewModel()
+        {
+        }
+
+        public EditRegistrationPromptViewModel(CustomPrompt customPrompt)
+        {
+            PromptNumber = customPrompt.CustomPromptNumber;
+            Prompt = customPrompt.CustomPromptText;
+            Mandatory = customPrompt.Mandatory;
+            Options = customPrompt.Options;
+            OptionsString = NewlineSeparatedStringListHelper.JoinNewlineSeparatedList(customPrompt.Options);
+        }
+
+        public int PromptNumber { get; set; }
+
+        public string Prompt { get; set; }
+
+        public bool Mandatory { get; set; }
+
+        [MaxLength(4000, ErrorMessage = "The complete list of answers must be less than 4000 characters.")]
+        public string? OptionsString { get; set; }

```

Comment:
How is this displayed on the UI?

Reply:
![image](https://user-images.githubusercontent.com/59561751/119336212-45a44d80-bc85-11eb-8c1c-73a232654d62.png)

I've noticed I need to sort the word break

Reply:
Could we test this condition in the controller and add the error to the "Add a new answer" field (as the links in error summary need to refer to a visible input with an error message)? Maybe it's worth adding the max number of characters the final answer can be onto the error message, e.g.
The complete list of answers must be less than 4000 characters. The new answer can be maximum 5 characters long.

Reply:
Yeah I can change it to check it manually mark against the textbox instead. Would probably simplify some of the ModelState logic.